src/picongpu/include/particles/ionization/ionizationEnergies.param

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2014 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* This file contains the ionization energies for different species. */
+
+#pragma once
+
+// Due to the current state of non-operability of the code this file is not included, yet.
+
+/* << TODO >> 
+ * - Make compatible for multiple atom species
+ * - Probably do some calculation from energy to required field strength (or do that model dependent)
+ * - Introduce some clever way of storage, like matrices for different charge states of the same species
+ * - future: even different states of excitation should be possible */
+
+namespace picongpu
+{
+    
+    namespace SI
+    {
+    
+        // ionization energy for ground state hydrogen: 13.6 eV --> converted to Joule
+        const double IONIZATION_ENERGY_SI = 2.179e-18; // Joule = kg * m^2 / s^2
```

Comment:
That is really cool for me as non physicist. :+1: 
Please move the comment over this line.